node/overseer/src/lib.rs

```diff
@@ -1242,15 +1244,15 @@ where
 		CG: Subsystem<OverseerSubsystemContext<CollationGenerationMessage>> + Send,
 		CP: Subsystem<OverseerSubsystemContext<CollatorProtocolMessage>> + Send,
 	{
-		let (events_tx, events_rx) = mpsc::channel(CHANNEL_CAPACITY);
+		let (events_tx, events_rx) = metered::channel(CHANNEL_CAPACITY, "overseer_events");
```

Comment:
How exactly are these supposed to be used? It would probably be useful to automatically create a metric in the constructor for each channel created

Reply:
That was one option but I decided to allow having a `Meter` accessor that has access to the `Arc<Atomic*>` so snapshots can be made at arbitrary points in time.

Reply:
in any case probably useful logging/metrics of some sort would be really good within this PR, like the periodic metric updates in overseer loop or smething

Reply:
What do you have in mind? Just stats every 10secs or a histogram with ascii bars?

Reply:
just -per-1-sec prometheus metrics with a `Gauge`?

Reply:
Oh, I see

Reply:
I'll skip that for now, the histogram sampling should be enough and there would make the instantiation of a channel failable or the metric could not be registered and would be silently dropped. Not sure either is good.